.github/workflows/build.yml

```
@@ -0,0 +1,172 @@
+      - name: Checkout the Git repository
+        uses: actions/checkout@v6
+      - *install-emake
+      - name: Building sdist
+        run: emake build --sdist
+      - uses: actions/upload-artifact@v6
+        with:
+          name: pip-sdist
+          path: dist/*
+          if-no-files-found: error
+
+  build-any-wheel:
```

actionlint (1.7.12)

[error] job "build-any-wheel" needs job "test" which does not exist in this workflow

(job-needs)